apps/laboratory/tests/smart-account.spec.ts

```diff
@@ -79,8 +79,9 @@ testModalSmartAccount(
     await walletModalPage.openAccount()
     await walletModalPage.openSettings()
     await walletModalPage.togglePreferredAccountType()
+    await walletModalValidator.expectChangePreferredAccountToShow(EOA)
     await walletModalPage.disconnect()
-    await walletModalPage.page.waitForTimeout(500)
+    await walletModalPage.page.waitForTimeout(1000)
```

Comment:
We shouldn't do timeouts, can we find another way to delay? E.g. waiting for connect button to show "Connect"?